android/app/src/main/kotlin/coop/polypoly/polypod/FeatureCardAdapter.kt

```diff
@@ -55,13 +60,80 @@ class FeatureCardAdapter(
         view.setOnClickListener {
             // FIXME - navigation assumes we're coming from FirstFragment,
```

Comment:
That's the thing with fixme, another PR and still not fixed. Maybe raise a Jira ticket with it? Would fixing it affect authentication? Probably not, but just in case.